de.gonicus.gonnect.yml

```diff
@@ -0,0 +1,104 @@
+id: de.gonicus.gonnect
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: gonnect
+
+finish-args:
+  - --device=all
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pcsc
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.mpris.MediaPlayer2.*
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-music:ro
+  - --filesystem=xdg-download:rw
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --share=network
+  - --share=ipc
+
+modules:
+- name: kerberos
+  subdir: src
+  config-opts:
+    - --localstatedir=/var/lib
+    - --sbindir=${FLATPAK_DEST}/bin
+    - --disable-rpath
+    - --disable-static
+  cleanup:
+    - /bin
+    - /share/et
+    - /share/examples
+  post-install:
+    - install -Dm644 ../krb5.conf -t ${FLATPAK_DEST}/etc/
+  sources:
+    - type: archive
+      url: "https://kerberos.org/dist/krb5/1.21/krb5-1.21.3.tar.gz"
+      sha256: b7a4cd5ead67fb08b980b21abd150ff7217e85ea320c9ed0c6dadd304840ad35
+      x-checker-data:
+        type: anitya
+        project-id: 13287
+        stable-only: true
+        url-template: "https://kerberos.org/dist/krb5/${major}.${minor}/krb5-${major}.${minor}.${patch}.tar.gz"
+    - type: file
+      path: "krb5.conf"
+
+- name: pjsip
+  buildsystem: autotools
+  config-opts:
+  - --disable-video
+  - --enable-ext-sound
+  - CFLAGS=-fPIC -DPJ_HAS_IPV6=1
+  sources:
+  - type: git
+    url: https://github.com/pjsip/pjproject.git
+    tag: 2.15.1
+    commit: 7de6e686fee1642f5443a3f39f0e5ad5e478a117
+    x-checker-data:
+      type: git
+      tag-pattern: '^([0-9]+\.[0-9]+\.[0-9]+)$'
+
+- name: openldap
+  config-opts:
+    - --disable-static
+    - --disable-slapd
+    - --disable-slurpd
+    - --disable-bdb
+    - --disable-hdb
+    - --with-tls=openssl
+  cleanup:
+    - /bin
+  sources:
+    - type: archive
+      url: "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.9.tgz"
+      sha256: 2cb7dc73e9c8340dff0d99357fbaa578abf30cc6619f0521972c555681e6b2ff
+
+- shared-modules/libusb/libusb.json
+
+- name: hidapi
+  buildsystem: cmake-ninja
+  sources:
+  - type: git
+    url: https://github.com/libusb/hidapi.git
+    tag: hidapi-0.14.0
+    commit: d3013f0af3f4029d82872c1a9487ea461a56dee4
+    x-checker-data:
+      type: git
+      tag-pattern: '^(hidapi-[0-9.]+)$'
+
+- name: gonnect
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_MODULE_PATH=/app/lib/${FLATPAK_ARCH}-linux-gnu/cmake/Qt6
+  - -DCMAKE_LIBRARY_PATH=/usr/lib
+  - -DCMAKE_INSTALL_RPATH=/usr/lib
+  sources:
+  - type: git
+    url: https://github.com/gonicus/gonnect.git
+    tag: v1.0.2
+    commit: a57ee5bbffe11342ff79a88e93491a750be5b98e
```

Comment:
this is the wrong commit for the tag.

Reply:
Oups. That happens when doing things manually ;-) Fixed it.